app/templates/cookie_policy.html

```diff
@@ -11,6 +11,18 @@
     {% if migration_banner_enabled %}
       {{migration_banner()}}
     {% endif %}
+
+    <div id="govuk-notification-banner-id" class="govuk-notification-banner govuk-notification-banner--success js-cookies-page-success" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner" hidden>
```

Comment:
Could you make this ID more unique and meaningful please?

Reply:
changed ID to "cookie-setting-saved-banner"